applications/launchpad/gui-react/src/components/TBot/TBotPrompt/styles.ts

```diff
@@ -6,28 +6,29 @@ export const PromptContainer = styled(animated.div)<{ $floating?: boolean }>`
   right: 40px;
   bottom: 40px;
   z-index: 1;
-  width: ${({ $floating }) => ($floating ? '476px' : '100%')};
+  width: ${({ $floating }) => ($floating ? '476px' : '692px')};
```

Comment:
all these magic numbers give me anxiety ^^
(just a comment, no changes implied)

Reply:
I know what you mean. How about a follow-up issue to replace the magic numbers with padding / margin where possible? I'd prefer not get dragged into chasing this just now.

Reply:
yeah we can make it a tech debt task, that's totally cool